osu.Game/Online/API/OAuth.cs

```diff
@@ -39,8 +39,8 @@ internal OAuth(string clientId, string clientSecret, string endpoint)
 
         internal void AuthenticateWithLogin(string username, string password)
         {
-            if (string.IsNullOrEmpty(username)) throw new ArgumentException("Missing username.");
-            if (string.IsNullOrEmpty(password)) throw new ArgumentException("Missing password.");
+            ArgumentException.ThrowIfNullOrEmpty(username);
+            ArgumentException.ThrowIfNullOrEmpty(password);
```

Comment:
Please revert, exception messages are user-facing via `APIAccess.LastLoginError` (which is then read by `LoginForm`)

Reply:
Done